vm/src/obj/objstr.rs

```diff
@@ -110,6 +111,16 @@ pub fn get_value(obj: &PyObjectRef) -> String {
     }
 }
 
+pub fn get_value_as_ref(obj: &PyObjectRef) -> Ref<str> {
```

Comment:
Maybe you could take a look here for some ideas? https://github.com/RustPython/RustPython/blob/master/vm/src/obj/objdict.rs#L27

Reply:
This is almost identical implementation (I only explicitly return the `Ref<str>` type, instead of `impl Deref<Target = str> + 'a`).

Reply:
Yes, you are right.